layers/meta-tegrademo/recipes-bsp/tegra-fitimage/tegra-fitimage_1.0.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "Install tegra fitImage in the location expected by u-boot"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+COMPATIBLE_MACHINE = "(cot)"
+
+do_install() {
+    install -d ${D}/boot
+    install -m 0644 ${DEPLOY_DIR_IMAGE}/fitImage ${D}/boot/fitImage-${MACHINE}
```

Comment:
@madisongh 
* I want to have the following lines instead 
```
root@jetson-tx2-devkit-cot:~# ls -lrta /boot/
total 13892
-rw-r--r--    1 root     root      14213662 Mar  9  2018 fitImage-4.9.201-l4t-r32.5+ge443483e8c9c
lrwxrwxrwx    1 root     root            40 Mar  9  2018 fitImage -> fitImage-4.9.201-l4t-r32.5+ge443483e8c9c
```
* With the actual implementation, I have the following lines:
```
root@jetson-tx2-devkit-cot:~# ls -lrta /boot/
total 35644
-rw-r--r--    1 root     root      39309146 Mar  9  2018 fitImage-jetson-tx2-devkit-cot
lrwxrwxrwx    1 root     root            30 Mar  9  2018 fitImage -> fitImage-jetson-tx2-devkit-cot
drwxr-xr-x    2 root     root          4096 Mar  9  2018 .
```

Reply:
Something like this:
```
    fitImgPath=$(readlink -f ${DEPLOY_DIR_IMAGE)/fitImage)
    fitImgName=$(basename "$fitImgPath")
    install -m 0644 $fitImgPath ${D}/boot/
    ln -sf $fitImgName ${D}/boot/fitImage
```

Reply:
Thanks @madisongh 
I will test it now and will let you know.

Reply:
I got this: 
```
drwxr-xr-x    2 root    root                        0 Apr  6  2011 /boot
lrwxrwxrwx    1 root    root                       77 Apr  6  2011 /boot/fitImage -> fitImage--4.9.201+git0+c68230c5b4-r0-jetson-tx2-devkit-cot-20210329095328.bin
-rw-r--r--    1 root    root                 39309146 Apr  6  2011 /boot/fitImage--4.9.201+git0+c68230c5b4-r0-jetson-tx2-devkit-cot-20210329095328.bin
```

Reply:
You're at least getting a fully-versioned name... is that not enough? That would at least provide a way to have multiple images installed in parallel, if needed.

You can look at the do_deploy task for the fitImage to get the naming scheme used there, and maybe adjust it so the canonical name matches, or is closer to, the naming scheme you're looking for. Just be careful about making any changes to the naming scheme.